From 47f1292116a83c4a760328e7675ccd50545d4e42 Mon Sep 17 00:00:00 2001
From: Matt Ellis <matell@microsoft.com>
Date: Fri, 28 Jul 2017 10:52:47 -0700
Subject: [PATCH] Do not build Microsoft.Packaging.Tools.Trimming

It is not used by the CLI, so there's no reason to build it.
---
 packages.builds | 1 +
 src.builds      | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages.builds b/packages.builds
index f9225b8..9f1bccc 100644
--- a/packages.builds
+++ b/packages.builds
@@ -4,6 +4,7 @@
   
   <ItemGroup>
     <Project Include="**/*.pkgproj" />
+    <Project Remove="Microsoft.Packaging.Tools.Trimming/pkg/Microsoft.Packaging.Tools.Trimming.pkgproj" />
   </ItemGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.traversal.targets))\dir.traversal.targets" />
diff --git a/src.builds b/src.builds
index 0a26dbc..bc3dc17 100644
--- a/src.builds
+++ b/src.builds
@@ -6,7 +6,6 @@
     <Project Include="netstandard/ref/*.csproj" />
     <Project Include="netstandard/src/*.builds" />
     <Project Include="platforms/dir.proj" />
-    <Project Include="Microsoft.Packaging.Tools.Trimming/tasks/*.builds" />
   </ItemGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.traversal.targets))\dir.traversal.targets" />
-- 
1.8.3.1

